samples/pt-br/Primeiros Passos/index.html

```diff
@@ -77,32 +77,32 @@
         
     <h3>Visualize alterações no HTML e CSS ao vivo no navegador</h3>
     <p>
-        Você sabe aquela "dança salvar/recarregar" que temos feito há anos? Aquela onde você faz mudanças no seu editor, clica em salvar, alterna para o navegador e então recarrega a página para finalmente ver o resultado? Com Brackets, você não precisa fazer essa dança.
+        Você sabe aquela "dança salvar/recarregar" que temos feito há anos? Aquela onde você faz mudanças no seu editor, clica em salvar, alterna para o navegador e então recarrega a página para finalmente ver o resultado? Com o Brackets, você não precisa fazer essa dança.
     </p>
     <p>
-        Brackets vai abrir uma <em>conexão ao vivo</em> com o seu navegador local e vai empurrar atualizações CSS enquanto você digita! Você já deve estar fazendo alguma coisa como esta hoje com ferramentas baseadas em navegador, mas com Brackets
+        O Brackets vai abrir uma <em>conexão ao vivo</em> com o seu navegador local e vai empurrar atualizações CSS enquanto você digita! Você já deve estar fazendo alguma coisa como esta hoje com ferramentas baseadas em navegador, mas com o Brackets
```

Comment:
"O Brackets vai abrir uma <em>conexão ao vivo</em> com o seu navegador local e vai **empurrar** atualizações CSS enquanto você digita!"

I think **empurrar** is a little bit out of context in here, maybe **enviar as atualizações no CSS**, what you guys think?